docs: Goose on Windows via WSL (#901)
block/goose: documentation/docs/getting-started/installation.md

  </TabItem>
</Tabs>

#### Installing on Windows via WSL

> While Goose Desktop and CLI doesn't work on Windows directly atm, It is possible to run Goose CLI on Windows using the Windows Subsystem for Linux (WSL).

Reviewer: ```suggestion
There isn't native installation support for Windows, however you can run Goose using WSL (Windows Subsystem for Linux).
```

> While Goose Desktop and CLI doesn't work on Windows directly atm, It may be possible to run Goose CLI on Windows using the Windows Subsystem for Linux (WSL).

Reviewer: may be possible? is it or isn't it? lol

Author: On it.

<details>

  <summary>Installing Goose on WSL</summary>

  #### Install WSL

>   Open PowerShell as Administrator and install WSL and the default Ubuntu distribution:

Reviewer: ```suggestion
  Open [PowerShell](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.5) as Administrator and install WSL and the default Ubuntu distribution:
```

Reviewer: doesnt hurt to link. if they had to make a guide, there's likely a use case for it

  ```bash
  wsl --install
  ```

  Restart your computer if prompted.

>   #### Update and install required packages

Reviewer: ```suggestion
  #### Update and Install Required Packages
```

>   Open the Ubuntu app from the start menu, complete the initial setup and update 

Reviewer: ```suggestion
  Open the Ubuntu app from the start menu, complete the initial setup and update.
```

  ```bash
  sudo apt update && sudo apt upgrade -y
  ```

>   #### Install Goose CLI

Reviewer: instead of repeating these steps, can we just link them to where this is already written? I'm trying to get us away from having duplicated text all throughout the docs
